src/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Leaderboard</title>
+</head>
+<body>
+  <div class="body-container">
+    <h1>Leaderboard</h1>
+    <div class="leaderboard-container">
+      <section class="recent-scores">
+        <div class="recent-scores-header">
+          <h2>Recent scores</h2>
+          <button>Refresh</button>
+        </div>
+        <ul id="leaderboard-list"></ul>
+      </section>
+      <section class="add-score">
+        <h2>Add your score</h2>
+        <form action="submit" id="add-form">
+          <input type="text" id="input-name" placeholder="Your name">
+          <input type="text" id="input-score" placeholder="Your score">
+          <input type="submit" value="Submit" id="button-submit">
+        </form>
+      </section>
```

Comment:
- [ ] Please, index.html main contain only root element. In this way, the page structure could be built on js code and index.html should have a `<div id="root"></div>` called on js code to add all that is here. Please, work on this. 🐶